### DISCUSSION TITLE
OSR下实现网页要素拖拽

### DISCUSSION THREAD

**Discussion by njuzhou** [CefView/QCefView#173] [Q&A]
问题描述:  在osr模式下，网页内部某要素支持拖拽至网页其他位置时，会出现无法拖拽的问题。

尝试方案: 在cefclient-Demo中，迁移osr_dragdrop_win.cc / osr_dragdrop_win.h / osr_dragdrop_events.h 三个文件，替换该工程下的CefViewBrowserClientDelegate.h中的startDragging方法，但发现仍然无效

不知作者大佬是否有意补充此功能

**Comment by tishion** [comment]
拖拽要基于Qt来实现，你直接使用cefclient里的是不行的，那个是基于win32的。

**Reply by njuzhou** [reply]
好吧，原来这样，我说咋一直用不了😂

**Reply by njuzhou** [reply]
感谢大佬回复
